pkg/transport/transport.go

```diff
@@ -64,7 +64,6 @@ func NewTransport(info TLSInfo, dialtimeoutd time.Duration) (*http.Transport, er
 }
 
 func (urt *unixTransport) RoundTrip(req *http.Request) (*http.Response, error) {
-	req2 := *req
-	req2.URL.Scheme = strings.Replace(req.URL.Scheme, "unix", "http", 1)
+	req.URL.Scheme = strings.Replace(req.URL.Scheme, "unix", "http", 1)
```

Comment:
just in case the *URL in the req will be used somewhere else,
```
url := *req.URL
req.URL = &url
req.URL.Scheme = strings.Replace(req.URL.Scheme, "unix", "http", 1)
```
?